src/Components/Autocomplete.fs

```diff
@@ -129,11 +143,33 @@ let createHelptextToolTip (overloads:DTO.OverloadSignature[]) (position:JQueryCo
 
         let! evt = Async.AwaitObservable helptextEvent.Publish
         match evt with
-        | Next by -> return! loop ((i + by + overloads.Length) % overloads.Length)
+        | Show text ->
+            return! async {
+                helptext.fadeOut() |> ignore
+                let! result = LanguageService.helptext text
+                return!
```

Comment:
I have a slight preference for writing:

    match result with
    | Some n ->
        return! foo 

Rather than writing:


    return!
        match result with
        | Some n ->
            foo 

I don't think it makes a difference here, but it would matter if you wanted to do more async operations in the body of the `match` - which isn't happening here at the moment.

(I also generally prefer to keep things on the left, probably because of my small laptop screen :-))